Documentation/packaging.md

```diff
@@ -25,32 +24,12 @@ wget, gpg, mktemp, md5sum, cpio, gzip, unsquashfs, sort
 
 patch, bc
 
-#### Additional build-time dependencies for the src flavor
+### Additional build-time dependencies for the src flavor
 
 intltoolize, libtoolize and all systemd dependencies.
 The dependencies may differ depending on the version of built systemd.
 
-## Run-time dependencies
-
-#### Basic run-time dependencies
-
-Linux configured with [suitable options](hacking.md#run-time-requirements)
-
-#### Additional run-time dependencies for the host flavor
-
-- systemd >= v222 with systemctl, systemd-shutdown, systemd, systemd-journald.
-- bash
-
-## Ownership and permissions of rkt directories
-
-In general, subdirectories of `/var/lib/rkt` should be created with the same ownership and permissions as done by `rkt install`, see [directory list](https://github.com/coreos/rkt/blob/master/rkt/install.go#L44).
-
-The package should create a group `rkt` and `/var/lib/rkt` should belong to group `rkt` with the `setgid` bit set (`chmod g+s`)
-
-When the ownership and permissions of `/var/lib/rkt` are set up correctly, users member of `rkt` should be able to fetch ACIs without using sudo.
-However, `root` is still required to run pods.
-
-## Offline builds
+### Offline builds
```

Comment:
"Offline builds" should have 2 `#`, otherwise it is inside the "Build-time dependencies" section.